assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -160,6 +162,10 @@ const baseActions = {
 		};
 
 		const { response, error } = yield fetchCreateAccountStore.actions.fetchCreateAccount( data );
+		if ( error ) {
+			yield registry.dispatch( STORE_NAME ).receiveError( error, 'createAccount', data );
```

Comment:
Thanks @felixarntz! Actually, i think the correct call should be just `.receiveError( error, 'createAccount' );` since we don't pass anything into `createAccount`. Then in a component, we will just call `.getErrorForAction( 'createAccount' )` instead of `.getErrorForAction( 'createAccount', [] );`.

The code is updated.

Reply:
@eugene-manuilov We should pass the empty array here, there should always be an `args` array. Even if you don't pass any arguments to a function, `args` should be an array essentially (which makes sense because that would also be the result if you did `...args` on such a function). Let's stick to the patterns Gutenberg uses here: https://github.com/WordPress/gutenberg/blob/master/packages/data/src/namespace-store/metadata/selectors.js

What we could do to simplify `getErrorForAction` and `getErrorForSelector` is to provide a default `args = []`. This way, if an action or selector is parameter-less, you don't need to provide `args` which makes sense. It should still technically be keyed with `args` always an array though.

Reply:
I like it. Updated.